docs/content/FAQ/_index.md

```diff
@@ -89,6 +109,21 @@ The log enables you to see which requests were requested from the server and ext
 2019-07-23 10:52:24,027 - report.py:580 - get_domain_issues() - INFO - Domain issues generated for reporting
 ```
 
+#### How do I change the log level of the Monkey Island logger?
```

Comment:
I feel like this section shouldn't split up "How do I find island logs" from "How do I find monkey logs". Fetching logs is one of the most important info we want users to know. Changing log level is an expert feature, most users don't need.